packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -64,6 +75,20 @@ function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): void {
   const task = event.execution.task;
   if (task.scope !== undefined && isTeamsfxTask(task)) {
     allRunningTeamsfxTasks.delete({ source: task.source, name: task.name, scope: task.scope });
+  } else if (isNpmInstallTask(task)) {
+    activeNpmInstallTasks.delete(task.name);
+    // when the task in the active terminal is ended.
+    // TODO: only when the task is ended successfully.
+    if (vscode.window.activeTerminal?.name === task.name) {
+      for (const hiddenTaskName of activeNpmInstallTasks) {
+        // show the first hiden terminal.
+        const hiddenTerminal = vscode.window.terminals.find(t => t.name === hiddenTaskName);
```

Comment:
will `vscode.window.terminals` be undefined? or at least it's an empty array?

Reply:
It will be an empty array if there's no opened terminals.